bugtool/configuration/bpftool.go

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package configuration
+
+import (
+	"fmt"
+	"strings"
+
+	dump "github.com/cilium/cilium/bugtool/dump"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const bpftoolMapDumpPrefix = "bpftool-map-dump-pinned-"
+
+// GenerateBPFToolTasks returns all tasks related to bpftool, including:
+// * Dumping pinned bpf maps (using bpftool).
+// * Listing bpf maps/programs/network-iface attachments.
+func GenerateBPFToolTasks() dump.Tasks {
+	ts := dump.Tasks{
+		newBPFMapTask("map", "show"),
+		newBPFMapTask("prog", "show"),
+		newBPFMapTask("net", "show"),
```

Comment:
These last two are in the `bpfmaps` directory but they are not about BPF maps.

Reply:
Switched it to be: `bpfmaps` -> `bpf` with a map sub-directory